src/main/java/org/broadinstitute/hellbender/Main.java

```diff
@@ -115,6 +115,15 @@ public Object instanceMain(final String[] args, final List<String> packageList,
      * @return the result of running  {program} with the given args, possibly null
      */
     protected static Object runCommandLineProgram(CommandLineProgram program, String[] rawArgs) {
+
+        /**
+         * The very first thing that any GATK application does is forces the JVM locale into US English, so that we don't have
+         * to think about number formatting issues.
+         *
+         * Placing the call here so that it is enabled for integration tests as well.
+         */
+        Utils.forceJVMLocaleToUSEnglish();
```

Comment:
I agree with @cmnbroad that this call should happen in a static initializer block in `Main.java`, which was the case in GATK3.